src/eko/evolution_operator/__init__.py

```diff
@@ -195,7 +195,9 @@ def quad_ker(
         )
         # scale var B is applied on the kernel
         if sv_scheme == sv.scheme_B:
-            ker = b.singlet_variation(ker, gamma_singlet, a1, order, nf, L)
+            ker = np.ascontiguousarray(ker) @ np.ascontiguousarray(
```

Comment:
If I understood correctly `np.ascontiguousarray` is just memory optimization: why do you need it here?
(I'd believe it to be more useful when you apply the function once, and you reuse the return value over and over)

Reply:
I don't remember the exact circumstances, but I do remember, that I definitely needed that here: https://github.com/N3PDF/eko/blob/e98971aa2e6457cfd4bb834cf6192a79f5a4aa07/src/eko/kernels/singlet.py#L338
(and similar places) otherwise numba wouldn't work correctly - and I also seem to remember that it was non-trivial to find ...

Reply:
yeah, so numba complains when you do the product of two matrices... Now I added a minimal fix.